.github/workflows/windows.yml

```diff
@@ -22,22 +22,42 @@ jobs:
         shell: bash
       - run: source activate && mingw32-make
         shell: bash
-      - run: mkdir jou
-      - name: Copy files
+      # We don't need to copy all of mingw64. We only need the linker.
+      # The less we copy, the smaller the resulting zip becomes.
+      - name: Copy some parts of mingw64 to jou/mingw64
+        shell: bash
+        run: |
+          # .a files compress well, so copy them all.
+          # CRT (C RunTime) files are needed because the linker implicitly adds them to every executable.
```

Comment:
```suggestion
          # CRT (C RunTime) files are needed because the linker implicitly adds them to every executable.
          # version_info.txt is only for curious humans.
```